docs/about/compatibility/openmp.md

```diff
@@ -1,4 +1,4 @@
-# OpenMP support in ROCm
+# OpenMP Support in ROCm
```

Comment:
I think the guideline is to keep everything in sentence case. Please check the internal style guide.